style.css

```diff
@@ -53,6 +59,39 @@ header {
   color: inherit;
 }
 
+.mobile-menu-close-icon {
+  color: #fff;
+  align-self: flex-end;
+  padding: 0;
+}
+
+.mobile-menu--hide {
+  display: none;
+  width: 0;
+  height: 0;
+}
+
+.mobile-menu--show {
+  display: flex;
```

Comment:
- [ ] You have used `display: flex` 18 times which means you are repeating yourself a lot and that is bad CSS practice. I would suggest you create a class with display: flex in it and use it at the HTML components that you need 
![image](https://user-images.githubusercontent.com/62705947/215591435-a5e24085-5ec4-412f-9292-a0ac3d818eb8.png)